resources/metadata/abapAddonAssemblyKitCheckCVs.yaml

```diff
@@ -0,0 +1,68 @@
+metadata:
+    name: abapAddonAssemblyKitCheckCVs
+    description: This step checks the validity of Software Component Versions.
+    longDescription: |
+      This steps takes a list of Software Component Versions from the addonDescriptorFileName and checks whether they exist or are a valid successor of an existing Software Component Version.
+      It resolves the dotted version string into version, support package level and patch level and writes it to the commonPipelineEnvironment.
+
+spec:
+  inputs:
+    secrets:
+      - name: abapAddonAssemblyKitCredentialsId
+        description: Credential stored in Jenkins for Addon Assembly Kit as a Service System (AAKaaS)
+        type: jenkins
+    params:
+      - name: abapAddonAssemblyKitEndpoint
```

Comment:
Is this a full qualified url or just the endpoint? For URL we typically call the parameter `serverUrl`.

Reply:
That's a bit tricky - what we need as input for this parameter is: 
- In URI terms we just need **scheme and authority** - not the path
- in URL terms we just need the **scheme, host and port** - not the path, 
- the WSDL term endpoint refers usually just to the URL including the path which we do not need as we have the path hard coded because every step uses a different path and we just wanted to have a single parameter instead of 7 different ones - especially as all 7 steps must be called in the same AAKaaS "system"
... so all did not match 100%...